modules/fscloud/variables.tf

```diff
@@ -0,0 +1,84 @@
+variable "region" {
+  type        = string
+  description = "The IBM Cloud region where the Cloudant instance will be provisioned."
+  default     = "us-south"
+}
+
+variable "plan" {
```

Comment:
remove plan from here - it has to be standard for dedicated. hard code it in the main.tf of fscloud profile